metrics-operator/api/v1alpha3/keptnmetric_webhook_test.go

```diff
@@ -0,0 +1,78 @@
+package v1alpha3
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func TestKeptnMetric_validateRangeInterval(t *testing.T) {
+
+	tests := []struct {
+		name string
+		Spec KeptnMetricSpec
+		want *field.Error
+	}{
+		{
+			name: "with-wrong-interval",
+			Spec: KeptnMetricSpec{
+				Range: &RangeSpec{Interval: "5mins"},
+			},
+			want: field.Invalid(
+				field.NewPath("spec").Child("range").Child("interval"),
+				"5mins",
+				errors.New("Forbidden! The time interval cannot be parsed. Please check for suitable conventions").Error(),
+			),
+		},
+		{
+			name: "with-empty-interval",
+			Spec: KeptnMetricSpec{
+				Range: &RangeSpec{Interval: ""},
+			},
+			want: field.Invalid(
+				field.NewPath("spec").Child("range").Child("interval"),
+				"",
+				errors.New("Forbidden! The time interval cannot be parsed. Please check for suitable conventions").Error(),
+			),
+		},
+		{
+			name: "with-nil-range",
```

Comment:
Added test cases for:
- if `Range == nil` 
- if `Range` is not provided

Not sure if these are the same case, will remove `with-no-range` test case then. @bacherfl

Reply:
thanks for the additional tests - those two cases should be the same, so I think the `with-no-range` can be removed